app/controllers/users_controller.rb

```diff
@@ -0,0 +1,16 @@
+class UsersController < ApplicationController
+
+  def new
+    @user = User.new
+  end
+
+  def create
+   @user = User.new(user_params)
```

Comment:
You are not creating a user here, just instantiating it.
Also, you are not rendering anything or returning anything.